openlibrary/core/bookshelves_events.py

```diff
@@ -124,6 +140,23 @@ def select_distinct_by_user_type_and_year(cls, username, event_type, year):
 
         return list(oldb.query(query, vars=data))
 
+    @classmethod
+    def total_yearly_reading_goals(cls, since: date | None = None) -> int:
+        """Returns a Storage object of <Storage {'count': int}> where 'count' specifies the
+        number reading goals updated. `since` may be used
+        to limit the result to those reading goals updated since a specific
+        date. Any python datetime.date type should work.
+        :param since: returns all reading goals after date
+        """
+        oldb = db.get_db()
+
+        query = "SELECT count(*) from yearly_reading_goals"
+        if since:
+            query += " WHERE updated >= $since"
+        results = tuple(oldb.query(query, vars={'since': since}))
+        print('total_yearly_reading_goals', results)
```

Comment:
```suggestion
        results = list(oldb.query(query, vars={'since': since}))
```
While casting to a `tuple` works here, please cast to a `list` instead.  This is how it's done throughout the codebase, and I'm not sure if using `tuple` here may have unintended side-effects.

Also, remove the `print` statement.